test/run-tests.py

```diff
@@ -902,8 +903,8 @@ def main(args):
     variables = {}
     variables['test_dir'] = os.path.abspath(TEST_DIR)
     variables['bindir'] = options.bindir
-    variables['gen_wasm_py'] = find_exe.GEN_WASM_PY
-    variables['gen_spec_js_py'] = find_exe.GEN_SPEC_JS_PY
+    variables['gen_wasm_py'] = os.path.join(TEST_DIR, 'gen-wasm.py')
```

Comment:
do these refactorings have anything to do with the CI part?

Reply:
I'm gonna split them out.